MarsRover/Program.cs

```diff
@@ -8,7 +9,13 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var input = new ConsoleInput();
+            var output = new ConsoleOutput();
+            var mapInput = new FileInput();
+            var map = new Map(5, 4, new List<Square>()); // TODO: think this shows I need another class!

```

Comment:
Sounds like it! Potentially something that creates a map for you given some map input?